src/subdataframe/subdataframe.jl

```diff
@@ -1,154 +1,142 @@
 """
-    SubDataFrame{<:AbstractVector{Int}} <: AbstractDataFrame
+    SubDataFrame{<:AbstractDataFrame,<:AbstractIndex,<:AbstractVector{Int}} <: AbstractDataFrame
 
-A view of row subsets of an AbstractDataFrame
+A view of row subsets of an `AbstractDataFrame`.
 
-A `SubDataFrame` is meant to be constructed with `view`.  A
-SubDataFrame is used frequently in split/apply sorts of operations.
-
-```julia
-view(d::AbstractDataFrame, rows)
-```
+A `SubDataFrame` is meant to be constructed with `view` when a collection of
+rows and columns is selected.
+It is also created by some methods of [`by`](@ref) and [`combine`](@ref).
 
 ### Arguments
```

Comment:
This isn't relevant now that `view` isn't shown here.

Reply:
I have rewritten docstrings for `SubDataFrame` and `DataFrameRow`. Please give comments on them or just push a fix.